lib/MusicBrainz/Server/Email.pm

```diff
@@ -624,6 +625,34 @@ EOF
 
     my $email = $self->_create_email(\@headers, $body);
     $self->_send_email($email);
+
+    if ($opts{send_to_self}) {
+        my $copy_subject = 'Copy of your report of ' . $reported_user_name;
+
+        my @copy_headers = (
+            'To'          => _user_address($opts{reporter}),
+            'Sender'      => $EMAIL_NOREPLY_ADDRESS,
+            'Subject'     => _encode_header($copy_subject),
+            'Message-Id'  => _message_id('editor-report-copy-%s-%s-%d', $reporter->id, $reported_user->id, time),
+        );
+
+        push @copy_headers, 'From', _user_address($reporter, 1);
+
+        my $copy_body .= <<EOF;
+This is a copy of your report of MusicBrainz editor '$reported_user_name':
+------------------------------------------------------------------------
+$subject for the following reason:
```

Comment:
Can we reuse `$body` here? Or save the initial `my $body` to a separate var if you don't want to include the reply info?

Reply:
I think so, yes. Reordered the email a bit so that all the content (inc. the message) goes first, and the you-can-reply info goes last, since that makes copying it easier too.